introspection-engine/connectors/introspection-connector/src/lib.rs

```diff
@@ -22,6 +22,8 @@ pub trait IntrospectionConnector: Send + Sync + 'static {
     async fn introspect(
         &self,
         existing_data_model: &Datamodel,
+        config: &Configuration,
+        render_config: bool,
```

Comment:
would these two fit in `IntrospectionContext`? Since we are evolving that API, fewer arguments is better. ofc just a suggestion